packages/react-context-selector/README.md

```diff
@@ -0,0 +1,11 @@
+# @fluentui/react-context-selector
+
+**React Context Selector components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+
+These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+
+To import React Context Selector components:
+
+```js
+import { ComponentName } from '@fluentui/react-context-selector';
+```
```

Comment:
Can we move README from an existing package?

Reply:
Dammit thought I caught it before you reviewed 😋